index.html

```diff
@@ -302,12 +308,16 @@ <h3>Multi-Post Stories</h3>
 
           <br />
           <br />
-          <hr />
+          
         </div>
+
+<!--Social Media Navigators-->
         <div class="footer flame">
-          <!--Social Media Navigators-->
 
+          
+          
           <div class="social">
+          
             <a href="https://github.com/">
               <i class="fa fa-github fa-1x" aria-hidden="true"></i
             ></a>
```

Comment:
- [ ] I noticed the submit button and the social links section don't match with the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) and According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins), so kindly fix it 👌 

| Figma Design  | Your Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/177536036-917d320c-2c3e-4193-9e8e-a6a87349aa6e.PNG" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/177536046-4eb0075f-15f7-40b9-bb6c-4815ebbaae38.PNG" width="500" /> |